### PR TITLE
Fix typos and add more comments/documentation

### LINKED ISSUE #101: Typos in documentation
Hello,

I've noticed there are couple copy-paste typos in documentation, resp. in documentation of `gc.go`.

Example:
```
    // SetLineJoin sets the current line join
    SetLineJoin(join LineJoin)
    // SetLineJoin sets the current dash
    SetLineDash(dash []float64, dashOffset float64)
    // ^^^ it should say SetLineDash
```

I can create a PR for those. May be this is a duplicate of #50?

### PR DESCRIPTION
**Pull request by zstyblik**
Commit fixes couple typos and adds more documentation to
GraphicContext interface in `gc.go`.

Fixes #101

### CODE REVIEW

File: gc.go [hunk 1]
```diff
@@ -37,27 +38,43 @@ type GraphicContext interface {
...
 	SetFontSize(fontSize float64)
+	// GetFontSize gets the current font size
 	GetFontSize() float64
+	// SetFontData sets the current FontData
 	SetFontData(fontData FontData)
+	// GetFontData gets the current FontData
 	GetFontData() FontData
+	// DrawImage draws the raster image in the current canvas
 	DrawImage(image image.Image)
 	Save()
```
Comment (llgcode) [inline gc.go]: Save the context and push it to the context stack

File: gc.go [hunk 2]
```diff
@@ -37,27 +38,43 @@ type GraphicContext interface {
...
 	Restore()
```
Comment (llgcode) [inline gc.go]: Restore remove the current context and restore the last one

File: gc.go [hunk 3]
```diff
@@ -37,27 +38,43 @@ type GraphicContext interface {
...
+	// Clear fills the current canvas with a default transparent color
 	Clear()
 	ClearRect(x1, y1, x2, y2 int)
```
Comment (llgcode) [inline gc.go]: ClearRect fills the specified rectangle with a default transparent color

**Comment by llgcode** [comment]
@zstyblik I've annotated the filewith some new comment. thx

**Comment by zstyblik** [comment]
@llgcode cool. I've pushed updated version.

**Comment by llgcode** [comment]
thanks @zstyblik

**Pull request merged**